graph/formats/dot/internal/lexer/lexer_test.go

```diff
@@ -52,3 +54,35 @@ func TestParseFile(t *testing.T) {
 		}
 	}
 }
+
+func TestParseFuzz(t *testing.T) {
+	r, err := zip.OpenReader("../../fuzz/corpus.zip")
+	if err != nil {
+		if os.IsNotExist(err) {
+			t.Skip("no corpus")
```

Comment:
How does `t.Skip` work? Would you not have to add a `return` statement after this?

Reply:
The return is not necessary. `Skip` works like `Fatal` it finishes the running goroutine.